python-sdk/src/astro/sql/operators/raw_sql.py

```diff
@@ -69,6 +96,52 @@ def make_row_serializable(rows: Any) -> Any:
             return [SdkLegacyRow.from_legacy_row(r) if isinstance(r, SQLAlcRow) else r for r in rows]
         return rows
 
+    @staticmethod
+    def results_as_list(result: ResultProxy) -> list:
+        """
+        Convert the result of a SQL query to a list
+        """
+        return cast(List[Any], result.fetchall())
+
+    @staticmethod
+    def results_as_pandas_dataframe(result: ResultProxy) -> pd.DataFrame:
+        """
+        Convert the result of a SQL query to a pandas dataframe
+        """
+        return pd.DataFrame(result.fetchall(), columns=result.keys())
+
+    def get_results_format_handler(self, results_format: str):
+        return getattr(self, f"results_as_{results_format}")
+
+    def get_handler(self):
+        # if a user specifies a results_format, we will return the results in that format
+        # note that it takes precedence over the handler
+        if self.results_format:
+            self.handler = self.get_results_format_handler(results_format=self.results_format)
+        if self.handler:
+            self.handler = self.get_wrapped_handler(
+                fail_on_empty=self.fail_on_empty, conversion_func=self.handler
+            )
+        return self.handler
```

Comment:
We should document this behaviour in docstrings as well as changelog so users know about this.

cc @phanikumv

Reply:
Looks like that is documented in docstirgs in L208 already -- we should duplicate it for the Operator docstrings too and have explicit docs for it in https://astro-sdk-python.readthedocs.io/en/stable/astro/sql/operators/raw_sql.html

Reply:
I'll add a ticket for this as well, so that this doesn't slip.

Reply:
We should also add a log line in the code so we users can know what took precedence by looking at logs

Reply:
Added log.